FINGERPRINT_Toolbox_Overview.adoc

```diff
@@ -87,6 +87,17 @@ Presentation Attack Detection includes the analysis of information from the regi
 
 Fingerprint PAD testing is a four step process. An image of the target fingerprint must first be acquired and the resultant image processed for mold making. Then the mold is made following the appropriate procedures for the mold type. If the mold is a direct impression of the target finger, the first step is eliminated. Once the mold is ready the cast material is prepared as necessary and applied to the mold. Upon removal from the mold the cast may be used as is or be further processed by applying selected coatings to enhance the physical properties of the PAI. Finally, the PAI is presented to the TOE.
 
+=== Common Testing Hygiene
+As the fingerprint sensors being tested here are all based on touch, several common actions should be taken to ensure the highest quality on the original samples. These actions can be divided into ensuring environmental clarity and minimizing noise. These actions will provide for the creation of better PAI with less need to "process" the samples using image enhancement (beyond what is needed to produce the PAI).
+
+==== Component Cleaning
+Component cleaning is to ensure environmental clarity for the presentation of any fingerprint (live or PAI). To the extent possible, all surfaces should be cleaned for each use (some PAI do not allow for explicit cleaning once created, but can be handled with proper care to ensure cleanliness). When a sample is take, both the finger in use and the material/sensor being used to record the fingerprint must be cleaned. Before a mold is used to create a PAI, it must be cleaned (according to the best practices for the materials in use). When a PAI is to be used, the PAI and the sensor must be cleaned (the PAI shouold be cleaned where possible, or handled in a way to ensure cleanliness when it isn't possible to be cleaned).
```

Comment:
> the material/sensor being used to record the fingerprint must be cleaned

Can we add something about cleaning "to the manufacturer's recommendations"? Cleaning certain types of sensors with household cleaning agents will deteriorate the platen.

Reply:
I have added the sentence 

> Sensors/scanners should be cleaned according to the manufacturer's recommendations to ensure proper functionality.